### ISSUE TITLE
Favorite checking is no progressing 

### ISSUE THREAD

**Issue by PaegelowM** [fmd-project-team/FMD#945]
Hello,

since a while thw favorite checking is not processing anymore.

it looks always like this and not more, so no new manga will be downloaded
![image](https://user-images.githubusercontent.com/53120853/80187685-5ad92380-8610-11ea-93cb-5d8ecb5ade5c.png)

this is my log
![image](https://user-images.githubusercontent.com/53120853/80187642-46952680-8610-11ea-96f7-763e3f7d0a7d.png)

does someone know how to solve this

already reinstalled fmd but no difference

**Comment by 7h3Cub3** [comment]
Does it block always onto the same manga in the favorite list ? tried to disable this manga to see if the problem is not with only this one ?

**Comment by 10lalib** [comment]
I have a similar problem.
Checking for updates on kissmanga, mangakakalot  manganelo doesn't work. And when i want to check the details for the magna (like number of chapters and so on it stays blank)
Btw this problem applies on all manga from the sources mentioned

**Comment by PaegelowM** [comment]
> Does it block always onto the same manga in the favorite list ? tried to disable this manga or site to see if the problem is not with only this one manga/site ?

i disabled Kissmanga and then fmd downloaded the other manga --> so it seems that the problem lies with Kissmanga

**Comment by 7h3Cub3** [comment]
> > Does it block always onto the same manga in the favorite list ? tried to disable this manga or site to see if the problem is not with only this one manga/site ?
> 
> i disabled Kissmanga and then fmd downloaded the other manga --> so it seems that the problem lies with Kissmanga

Yes sadly kissmanga has been very difficult lately, even copy pasting the cookies is often not enough to get it to work.
